docs/sources/datasources/prometheus/configure-prometheus-data-source.md

```diff
@@ -70,7 +75,8 @@ The first option to configure is the name of your connection:
 
 ### Connection section
 
-- **Prometheus server URL** - The URL of your Prometheus server. If your Prometheus server is local, use `http://localhost:9090`. If it is on a server within a network, this is the URL with port where you are running Prometheus. Example: `http://prometheus.example.orgname:9090`.
+- **Prometheus server URL** - The URL of your Prometheus server. {{< shared id="prom-data-source-url" >}}
+  If your Prometheus server is local, use `http://localhost:9090`. If it's on a server within a network, this is the URL with the port where you are running Prometheus. Example: `http://prometheus.example.orgname:9090`.
```

Comment:
Note this line is slightly different than https://github.com/grafana/website/pull/23522/files#diff-38b5be74cd826b998ea490ed06f121e10f6ce7ee4e027a0d15f88e1715c0d4d7, and this is what will show up in production.

Reply:
I think we're good. Thanks!